src/Symfony/Bridge/Twig/composer.json

```diff
@@ -22,7 +22,7 @@
     "require-dev": {
         "symfony/asset": "~2.7",
         "symfony/finder": "~2.3",
-        "symfony/form": "~2.7.8|~2.8,>2.8-BETA1",
+        "symfony/form": "~2.7,>=2.7.8",
```

Comment:
Since there are already a bunch of conflicts between 2.8 and 3.0 for composer.json files, what about updating all composer.json files to the caret operator on 3.0 instead?